android/app/build.gradle

```diff
@@ -76,11 +78,17 @@ android {
                 // Recommended by Google Play Store for testing
                 // https://developer.android.com/reference/tools/gradle-api/6.7/com/android/build/api/dsl/Ndk
                 debugSymbolLevel 'SYMBOL_TABLE'
-                // Flutter doesn't support x86, so we need add only supported platforms: https://docs.flutter.dev/deployment/android#what-are-the-supported-target-architectures
-                abiFilters 'arm64-v8a', 'armeabi-v7a', 'x86_64'
             }
         }
     }
+
+    applicationVariants.all { variant ->
+        variant.outputs.each { output ->
+            def abiCodes = ['armeabi-v7a': 1, 'arm64-v8a': 2, x86: 3, x86_64: 4]
```

Comment:
x86 is not supported by flutter

Reply:
That is just a generic mapping from the ABI code to a versionCode offset. We can remove 'x86' as it is of no effect here.